src/client/components/responsive-video/responsive-video.stories.js

```diff
@@ -17,14 +18,85 @@ storiesOf('Responsive video', module)
           "height": 360,
           "provider": "vimeo",
           "providerUid": "265601385",
-          "thumbnailUrl": "https://i.vimeocdn.com/video/695812178_640.jpg"
+          "thumbnailUrl": "https://i.vimeocdn.com/video/695812178_640.jpg",
+          "autoplay": false,
+          "loop": false,
         }
       }
     },
+    i18n: new VueI18n({
+      locale: 'en',
+    })
   }))
-  .add('YouTube', () => ({
+  .add('Vimeo with autoplay', () => ({
     components: { ResponsiveVideo },
-    template: '<responsive-video :video="video" />',
+    template: '<responsive-video :video="video" :autoplay="video.autoplay" :loop="video.loop" :mute="video.autoplay" />',
+    data() {
+      return {
+        video: {
+          "url": "https://vimeo.com/265601385",
+          "title": "A New Campaign - PickUp10",
+          "width": 640,
+          "height": 360,
+          "provider": "vimeo",
+          "providerUid": "265601385",
+          "thumbnailUrl": "https://i.vimeocdn.com/video/695812178_640.jpg",
+          "autoplay": true,
+          "loop": false,
+        }
+      }
+    },
+    i18n: new VueI18n({
+      locale: 'en',
+    })
+  }))
+  .add('Vimeo with loop and autoplay', () => ({
+    components: { ResponsiveVideo },
+    template: '<responsive-video :video="video" :autoplay="video.autoplay" :loop="video.loop" :mute="video.autoplay" />',
+    data() {
+      return {
+        video: {
+          "url": "https://vimeo.com/265601385",
+          "title": "A New Campaign - PickUp10",
+          "width": 640,
+          "height": 360,
+          "provider": "vimeo",
+          "providerUid": "265601385",
+          "thumbnailUrl": "https://i.vimeocdn.com/video/695812178_640.jpg",
+          "autoplay": true,
+          "loop": true,
+        }
+      }
+    },
+    i18n: new VueI18n({
+      locale: 'en',
+    })
+  }))
+  .add('YouTube default', () => ({
+    components: { ResponsiveVideo },
+    template: '<responsive-video :video="video" :autoplay="video.autoplay" :loop="video.loop" :mute="video.autoplay" />',
+    data() {
+      return {
+        video: {
+          "url": "https://www.youtube.com/watch?v=ZPIIT6siIaE&list=PL8HcoiB7g3NlaU-UU1boptT4tkz3KOXT2",
+          "title": "Een surfer neemt het op tegen de plastic vervuiling",
+          "width": 480,
+          "height": 270,
+          "provider": "youtube",
+          "providerUid": "ZPIIT6siIaE",
+          "thumbnailUrl": "https://i.ytimg.com/vi/ZPIIT6siIaE/hqdefault.jpg",
+          "autoplay": false,
+          "loop": false,
```

Comment:
Please define this the way it should come from dato, ie do not nest loop, autoplay, mute and title in the video property.